packages/http-client-csharp/eng/scripts/Get-CadlRanch-Coverage.ps1

```diff
@@ -63,6 +63,17 @@ foreach ($directory in $directories) {
     if ($LASTEXITCODE -ne 0) {
         exit $LASTEXITCODE
     }
+
+    # build the generated project
+    Write-Host "Building $subPath" -ForegroundColor Cyan
+    Get-ChildItem -Path $outputDir -Recurse -Filter '*.csproj' | ForEach-Object {
+        $command = "dotnet build $_"
+        Invoke $command
+        # exit if the build failed
+        if ($LASTEXITCODE -ne 0) {
+            exit $LASTEXITCODE
+        }
+    }
```

Comment:
here we added a step to build the solution to ensure the source code is correct.